src/current/v23.2/set-up-physical-cluster-replication.md

```diff
@@ -0,0 +1,309 @@
+    1  | system   | ready      | shared
+    2  | template | ready      | none
+    (2 rows)
+    ~~~
+
+### Create a user for the standby cluster
+
+If you would like to access the DB Console to observe your replication, you will need to create a user:
+
+1. Create a user:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE USER {your username} WITH LOGIN PASSWORD {'your password'};
+    ~~~
+
+1. To observe the replication activity, your user will need [`admin`]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role) privileges:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    GRANT admin TO {your username};
+    ~~~
+
+    Open the DB Console in your web browser: `https://{your IP or hostname}:8080/`, where you will be prompted for these credentials. Refer to [Physical Cluster Replication Monitoring]({% link {{ page.version.version }}/physical-cluster-replication-monitoring.md %}) for more detail on tracking relevant metrics for your replication stream.
+
+## Step 3. Verify certificates
+
+At this point, you have a primary and a standby cluster running. The next step is to verify that the clusters can communicate. Depending on how you manage certificates, you must ensure that all nodes on the primary and the standby cluster have access to the certificate of the other cluster.
+
+For example, if you followed the [Deploy CockroachDB]({% link {{ page.version.version }}/deploy-cockroachdb-on-premises.md %}) prerequisite, you need to add the `ca.crt` from the primary cluster to the `certs` directory on all the nodes in the standby cluster.
+
+1. Name the `ca.crt` from the primary cluster to a new name on the standby cluster. For example, `ca_primary.crt`.
+1. Place `ca_primary.crt` in the `certs` directory of the standby cluster nodes.
+
+You need to add the `ca.crt` from the standby cluster to the `certs` directory on all the nodes in the primary cluster.
+
+1. Name the `ca.crt` from the standby cluster to a new name on the primary cluster. For example, `ca_standby.crt`.
+1. Place `ca_standby.crt` in the `certs` directory of the primary cluster nodes.
+
+## Step 4. Start replication
+
+The system interface in the standby cluster initiates and controls the replication stream by pulling from the primary cluster. In this section, you will connect to the primary from the standby to initiate the replication stream.
+
+1. From the **standby** cluster, use your connection string to the primary.
+
+    The connection string contains:
+    - The replication user and password that you [created for the primary cluster](#create-a-replication-user-and-password).
+    - The node IP address or hostname of one node from the primary cluster.
+    - The path to the primary node's certificate on the standby cluster.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE VIRTUAL CLUSTER standbyapplication
+    FROM REPLICATION OF application
```

Comment:
I believe we'll want the user to also specify the template:

```
CREATE VIRTUAL CLUSTER standbyapplication LIKE template
FROM REPLICATION...`

Reply:
@stevendanna The `LIKE template` syntax is not required though, correct? I can include it. 

I'll also have the SQL pages up soon that will include the SQL railroad diagrams and more details on the syntax options for users.

Reply:
It's not _strictly_ required, but it helps make sure that the tenant that gets created has the correct "Capabilities."  If they don't do this, they'll need to run various follow-up commands to make sure their virtual cluster works the same as the one on the primary side.

Reply:
We have a thread in Slack ongoing re `capabilities`, so I will keep an eye on that and include further detail as needed. For now, added a placeholder sentence. 

I did try to use the `LIKE` syntax on my test clusters... when you specify the `template` are we talking about referring to the VC template from the primary? I received an error `ERROR: retrieving LIKE tenant record: tenant "application" does not exist` when trying to do that. Maybe that's because the app VC on the primary was already shared? 

However, I have a couple of VCs hanging about on the standby, so I tried referring to a diff VC on the standby with `LIKE` when creating a new replication job (and a new standby VC). That worked fine. But, I don't know if that's the point of `LIKE`.

Reply:
When you create the target cluster using the configuration profile `replication-target` it should create a virtual cluster named `template` in the destination as well that should work for this command on the destination.